charts/apisix-ingress-controller/templates/rbac.yaml

```diff
@@ -157,5 +153,6 @@ roleRef:
   name: {{ .Release.Name }}-clusterrole
 subjects:
   - kind: ServiceAccount
-    name: {{ .Release.Name }}-serviceaccount
+    name: {{ include "apisix-ingress-controller.serviceAccountName" . }}
```

Comment:
The name of service account should be opaque to the administrator, i don't think it's necessary to customize it, {{ .Release.Name }}-serviceaccount is enough.

Reply:
I don't think so, service account may not be created, user can use existd name.
see [best practices for RBAC](https://helm.sh/docs/chart_best_practices/rbac/).

Reply:
Well, we only need  the permission that we need, and these permissions are defined in the matched rbac.yaml. Why people need to use an existing service account? Why we need to let users to configure other service accounts?
 it's an extra burden for an administrator.

Reply:
If we hava multi ingress controller, we can use an existing service account, it depends on the user's choice.

`serviceAccount.create` default is True, so users can not concern, it's not an extra burden.

Reply:
If we could, expose as few as possible items. The service account is well designed for the permission, it's not easy to let users mimic one, he/she needs to look the one we provided.

For multiple ingress controllers, i don't think it's useful to share the service account, basically these controllers will spread in different namespaces, while service account should be put in the same namespace with the deployment.

Reply:
I suggest not to do that, we can consider it in the future if the community really has this demand.￼

Reply:
helm chart of kong/traefik/nginx ingress controller alway provided this future, I think this is a general future.

Do anyone else have any ideas. cc @gxthrj

Reply:
I'd like to know some scenes. I prefer the MVP principle, if we don't know whether it's useful, just don't add it.

Reply:
Actually, we use a unified administrator service account for nginx ingress controller, so hope can custom service account